### PR TITLE
Adapt the removal of libvirtd for upgraded Tumbleweed

### PR DESCRIPTION
**Pull request by chcao**
In Tumbleweed, the libvirtd is already removed and replaced by virtqemud, to adapt this change in libvirtd.service testing on upgraded Tumbleweed, enable, start and check the virtqemud.service after upgrading to Tumbleweed.

- Related ticket: https://progress.opensuse.org/issues/133856
- Needles: n/a
- Verification run: 
- https://openqa.opensuse.org/tests/3531808#step/nested_vm#1/22
- https://openqa.opensuse.org/tests/3531809#step/nested_vm#1/22

### CODE REVIEW

File: lib/services/libvirtd.pm [hunk 1]
```diff
@@ -60,6 +60,25 @@ sub check_service {
     systemctl 'is-active libvirtd';
 }
 
+# libvirt is removed from Tumbleweed, virtqemud is the replacement, so for
```
Comment (lemon-suse) [inline lib/services/libvirtd.pm:63]: Change it to 'libvirtd removed ...'

File: lib/services/libvirtd.pm [hunk 2]
```diff
@@ -115,10 +134,17 @@ sub full_libvirtd_check {
...
-    common_service_action($pkg, $type, 'is-enabled');
-    common_service_action($pkg, $type, 'is-active');
+    if (!(($stage eq 'after') && (get_var("VERSION") eq "Tumbleweed"))) {
+	common_service_action($pkg, $type, 'is-enabled');
+	common_service_action($pkg, $type, 'is-active');
+    }
 
     if ($stage eq 'after') {
+	if (get_var("VERSION") eq "Tumbleweed") {
+	    enable_upgraded_service();
```
Comment (lemon-suse) [inline lib/services/libvirtd.pm:144]: Others looks good to me, just feel a bit strange why need 3 functions here and each one only invoke once, could we just reserve one function here?

File: lib/services/libvirtd.pm (hunk 1 above)
Comment (chcao) [reply lib/services/libvirtd.pm]: I updated it, thanks.

File: lib/services/libvirtd.pm (hunk 2 above)
Comment (chcao) [reply lib/services/libvirtd.pm]: Updated it to one function.

**Comment by chcao** [comment]
> And there is CI issue existed.

Yes, resolved the CI issue.

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**